PowerEditor/src/Notepad_plus.cpp

```diff
@@ -6214,12 +6214,10 @@ void Notepad_plus::launchFunctionList()
 
 		_pFuncList->setBackgroundColor(bgColor);
 		_pFuncList->setForegroundColor(fgColor);
+		_pFuncList->reload();

```

Comment:
This modification made a regression, step to reproduce:
1. Open 2 different xml documents - xml_1 & xml_2, xml_2 is current document
2. Open Function List Panel - in which xml_2 is loaded.
3. Open Project Panel 1, if it is not in the same docking window of Function List, drag it in it.
4. Switch to xml_1 from xml_2
5. Switch to  Function List from Project Panel 1 (by using mouse or shortcut). 

Result before applying this PR:
Function List loads xml_1 content - Correct

Result after applying this PR:
Function List remains xml_2 content - Incorrect

Conclusion: reload() shouldn't be here. It should be where it was.